packages/mail/templates/membership_owner_autopay_failed.html

```diff
@@ -0,0 +1,106 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <meta http-equiv="x-ua-compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width,initial-scale=1">
+    <!--[if gte mso 15]>
+    <xml>
+      <o:officedocumentsettings>
+        <o:allowpng />
+        <o:pixelsperinch>96</o:pixelsperinch>
+      </o:officedocumentsettings>
+    </xml>
+    <![endif]-->
+    <style type="text/css">
+      {{sg_font_faces}}
+    </style>
+    <style type="text/css">
+      p {
+        color:#282828;font-size:17px;line-height:24px;
+        {{sg_font_style_sans_serif_regular}}
+      }
+      p strong {
+        {{sg_font_style_sans_serif_medium}}
+      }
+    </style>
+  </head>
+  <body style="margin: 0; padding: 0; background-color: #fff;">
+    <table border="0" cellpadding="0" cellspacing="0" width="100%">
+      <tbody>
+        <tr>
+          <td align="center" valign="top">
+            <table align="center" border="0" cellpadding="0" cellspacing="0" width="100%" style="max-width:640px;color:#282828;font-size:17px;line-height:24px;{{sg_font_style_sans_serif_regular}}">
+              <tbody>
+                <tr>
+                  <td style="padding:20px;">
+                    <p>
+                      Sehr geehrte Dame<br />
+                      Sehr geehrter Herr
+                    </p>
+                    {{#if `attempt_number == 3`}}
+                      <p>(attempt_number 3)</p>
+                      <p>Ihre Mitgliedschaft erneuerte sich am {{end_date}} um ein weiteres Jahr. Bedauerlicherweise konnten wir erneut den Jahresbeitrag von {{autopay_total}} Ihrer hinterlegten {{autopay_card_brand}}-Kreditkarte mit den Endziffern {{autopay_card_last4}} nicht belasten.</p>
+                      <p>Wir unternehmen am {{attempt_next_at}} einen letzten Versuch und bitten Sie, bis dahin <a href="{{prolong_url}}">Ihre Kreditkartendaten hier zu überprüfen</a> und gegebenenfalls zu aktualisieren.</p>
```

Comment:
```suggestion
                      <p>Wir unternehmen am {{attempt_next_at}} einen letzten Versuch und bitten Sie, bis dahin <a href="{{prolong_url}}">Ihre Kreditkartendaten hier gegebenenfalls zu aktualisieren</a>.</p>
```
Man kann die Daten nicht wirklich überprüfen. Die letzten vier Ziffern stehen ja schon oben.